appveyor/install-miniconda.ps1

```diff
@@ -76,8 +76,12 @@ $env:PATH = "${env:PYTHON};${env:PYTHON}\Scripts;" + $env:PATH
 
 # Conda config
 conda config --set always_yes true
-conda config --add channels astropy-ci-extras
 
+if ($conda_channels -ne $null) {

```

Comment:
so what about copying over the logic from ``travis/setup_dependencies_common.sh``. If it's not given set it to be equal of ``astropy-ci-extras astropy``, and then loop over and add all the channels.